docs/guides/k8s-monitoring-via-vm-cluster/README.md

```diff
@@ -414,9 +414,47 @@ The expected output is:
 ```text
 vmagent-victoria-metrics-agent-69974b95b4-mhjph                1/1     Running   0          11m
 ```
+## 4. Install vmauth from the Helm chart (optional)
 
+We can also deploy `vmauth` service in cluster. It can work as a gateway, distributing requests to multiple `vminsert`s and `vmselect`s. Key capabilities include load balancing and failover. See [vmauth](https://docs.victoriametrics.com/victoriametrics/vmauth/).
 
-## 4. Install and connect Grafana to VictoriaMetrics with Helm
+Deploying vmauth is optional. But if you need to build the [Multi-AZ](https://docs.victoriametrics.com/guides/vm-architectures/#multi-cluster-and-multi-az) or [Hyperscale](https://docs.victoriametrics.com/guides/vm-architectures/#the-hyperscale-cell-based) cluster, it is highly recommended to deploy `vmauth` as the entry point for the cluster.
```

Comment:
Let's keep the reader focused on the current setup size to avoid context swithcing. We can mention other sizes at the end of the document.